openlibrary/i18n/fr/messages.po

```diff
@@ -619,16 +639,83 @@ msgstr "Quelle page regardiez vous ?"
 msgid "Send"
 msgstr "Envoyer"
 
-#: EditButtons.html:23 EditButtonsMacros.html:26 account/create.html:86
-#: account/notifications.html:59 account/password/reset.html:24
-#: account/privacy.html:56 admin/imports-add.html:28 books/add.html:84
-#: books/edit/addfield.html:87 books/edit/addfield.html:133
-#: books/edit/addfield.html:177 covers/add.html:78 covers/manage.html:51
-#: databarAuthor.html:66 databarEdit.html:13 lists/widget.html:340
-#: merge/authors.html:97 support.html:71
+#: EditButtons.html:23 EditButtonsMacros.html:26 ReadingLogDropper.html:167
+#: account/create.html:90 account/notifications.html:59
+#: account/password/reset.html:24 account/privacy.html:56
+#: admin/imports-add.html:28 books/add.html:108 books/edit/addfield.html:87
+#: books/edit/addfield.html:133 books/edit/addfield.html:177 covers/add.html:79
+#: covers/manage.html:52 databarAuthor.html:66 databarEdit.html:13
+#: merge/authors.html:109 support.html:71
 msgid "Cancel"
 msgstr "Annuler"
 
+#: trending.html:10
+#, fuzzy
+msgid "Now"
+msgstr "non"
```

Comment:
Is this the correct translation?

Reply:
No, this is definitely not the correct translation.

My suggestions would be "maintenant" or " en ce moment". 

Thank you :)

Reply:
✅ maintenant